src/ffi/mod.rs

```diff
@@ -796,991 +746,6 @@ extern "C" {
     fn fortify_source();
 }
 
-#[no_mangle]
```

Comment:
Ater removing this part, I have not managed to understand how are we handling this FFI-level logic with UniFFI? Could you explain it a little bit?

Reply:
Added to LLT-5022